api/src/test/java/io/grpc/GlobalInterceptorsTest.java

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright 2022 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.fail;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+import org.junit.Test;
+
+public class GlobalInterceptorsTest {
+
+  @Test
+  public void setInterceptorsTracers() throws Exception {
+    StaticTestingClassLoader classLoader =
```

Comment:
All of the tests have this common code (lines 32-34). Can this be refactored into `setUp` with `classLoader` that's common for the this test class?

Reply:
You don't need setUp. Just add a field and initialize it:
```java
private final StaticTestingClassLoader classLoader = new StaticTestingClassLoader(...);
```